packages/core/test/Cardano/types/TxSubmissionErrors.test.ts

```diff
@@ -7,4 +9,12 @@ describe('Cardano/types/TxSuubmissionErrors', () => {
     });
     expect(error).toBeInstanceOf(Cardano.TxSubmissionErrors.FeeTooSmallError);
   });
+  test('maps all errors from Ogmios', () => {
+    const ogmiosErrors = Object.keys(TxSubmission.errors).sort();
+    const sdkErrors = Object.keys(TxSubmissionErrors)
+      .filter((e) => e !== 'UnknownTxSubmissionError')
+      .map((e) => e.slice(0, Math.max(0, e.length - 5)))
```

Comment:
Took a while to understand that it's getting rid of 'Error'. A comment here would be nice